### PR TITLE
Prod Deploy

### PR DESCRIPTION
**Pull request by silasbaur**

### CODE REVIEW

File: src/components/crds-carousel/crds-carousel.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+   */
+  @Prop() bulletColor: string | "gray";
+  /**
+   * Specifies title of caorousel and changes carousel layout
+   */
+  @Prop() title: string;
+  /**
+   * Specifies title color
+   */
+  @Prop() titleColor: string;
```
Comment (j-brammer) [inline src/components/crds-carousel/crds-carousel.tsx:52]: These comments feel very extraneous with very little value. The variables names seem good enough that we don't really need these comments.

File: src/components/form/form/form.tsx [hunk 2]
```diff
@@ -57,13 +57,35 @@ export class Form {
...
+        };
+
+        if (!readOnly && lastSubmissionDate !== null) {
+          formio.setAlert(
+            'info',
+            `You're already on the list for this group or event as of ${signupDate}, but may sign up again. If you think we've made a mistake, reach out to 
+            support@crossroads.net for further assistance.` 
+          );
+        };
+
```
Comment (j-brammer) [inline src/components/form/form/form.tsx:88]: I'd advise against putting copy in the code. We have a mechanism for getting content blocks from Contentful in other Components

File: src/components/crds-carousel/crds-carousel.tsx (hunk 1 above)
Comment (silasbaur) [reply src/components/crds-carousel/crds-carousel.tsx:52]: These comments are for documentation on the [notes tab in storybook](https://deploy-preview-871--crds-components.netlify.app/?path=/story/carousel--crds-carousel-default). We have been trying to keep every component we create well documented and consistent.

File: src/components/profile/personal-info/crds-date-of-birth/crds-date-of-birth.tsx [hunk 3]
```diff
@@ -49,6 +49,7 @@ export class DateOfBirth {
 
   @Method()
   public async isValid() {
+    window["woot"] = this.inputElement;
```
Comment (j-brammer) [inline src/components/profile/personal-info/crds-date-of-birth/crds-date-of-birth.tsx:52]: Oh good lord, this is my goof, Ill fix it

File: src/components/crds-carousel/crds-carousel.tsx (hunk 1 above)
Comment (j-brammer) [reply src/components/crds-carousel/crds-carousel.tsx:52]: Ah, so it's some feature we haven't been leveraging. Alright. thanks for the info

**Pull request merged**